administrator/components/com_joomgallery/tables/joomgalleryconfig.php

```diff
@@ -40,6 +40,13 @@ class TableJoomgalleryConfig extends JTable
   var $jg_checkupdate;
   var $jg_filenamewithjs;
   var $jg_filenamereplace;
+  var $jg_replaceimgtitle;
```

Comment:
Die neuen Optionen sollten auch in der **_administrator\components\com_joomgallery\helpers\config.php_** eingetragen werden.